### PR TITLE
Copy edits (1st pass)

### LINKED ISSUE #6: LED indicators
There is no diagram dedicated to showing the location of all the LEDs.  Instead, they are shown in Figure 3.1 alongside communication interfaces and in Figure 4.1.  Some of the LEDs shown in the figures are not described (e.g. GATE ENABLE LED, Status RGB LED and Communication LED), and the Inverter enabled LED is described but not shown.

### PR DESCRIPTION
**Pull request by clyde-johnston**
I have used the same text as Myxa wherever possible.  Please review.

### CODE REVIEW

File: docs/datasheet/Komar.tex [hunk 1]
```diff
@@ -15,50 +15,53 @@
...
-use in propulsion systems of light unmanned aerial (UAV), underwater, and surface vehicles. State-of-the-art vector
-control algorithms make Komar one of the most energy-efficient ESC on the market. Komar is compatible with virtually
-any PMSM or BLDC motor as it is able to measure the parameters of the driven motor and automatically optimize
-itself for best performance. Komar is based on \mbox{mitochondrik} control module.
-All hardware is open source and available on Github\footnote{\url{https://github.com/Zubax/Komar}}.
+Zubax Komar is a high-quality FOC ESC based on the \text{T\'elega} motor control technology. Komar is designed for
+use in propulsion systems of light unmanned aerial vehicles (UAV), unmanned underwater vehicles (UUVs) and
+unmanned surface vehicles (USVs). State-of-the-art vector control algorithms make Komar one of the most
+energy-efficient ESCs on the market. Komar is compatible with almost all PMSM and BLDC motors and continuously
+measures the motor parameters to optimize its own performance. Komar is based on the Mitochondrik control module,
```
Comment (dimracer) [inline docs/datasheet/Komar.tex:22]: should be the same as in the Myxa datasheet.

File: docs/datasheet/overview.tex [hunk 2]
```diff
@@ -1,27 +1,30 @@
...
+performance parameters including the controller temperature, the motor temperature, the instantaneous
+DC link voltage and instantaneous DC link current.
 
 \section{System integration}
-Zubax Komar is a single supply device, which means that the device does not expose any power supply inputs except
-for the high power supply. The 5 V rails of the CAN interfaces are not used by the device;
-rather, the device may provide 5 V power line for UAVCAN bus from its internal DC-DC converter if needed.
+Zubax Komar is a single-supply device that dedicates power to the motor. It does not expose any power supply
+inputs to its internal components and the 5\,V rails of the CAN interfaces cannot be used by the controller.
+Myxa does, however, offer a power delivery feature that when enabled will deliver 5\,V to the power
```
Comment (dimracer) [inline docs/datasheet/overview.tex:23]:
```suggestion
Komar does, however, offer a power delivery feature that when enabled will deliver 5\,V to the power
```

**Review by dimracer — CHANGES_REQUESTED** [review]

File: docs/datasheet/Komar.tex (hunk 1 above)
Comment (clyde-johnston) [reply docs/datasheet/Komar.tex:22]: I understood that Komar does not have an accessible UART interface, so I have removed this. Is this correct?

Comment (clyde-johnston) [reply docs/datasheet/Komar.tex:22]: Komar has this additional sentence:
>  Komar is based on the Mitochondrik control module, the hardware design of which is open source and available on Github.

Does this apply to Myxa too?  Or is this unique to Komar?

Comment (clyde-johnston) [reply docs/datasheet/Komar.tex:22]: Under "Applications"
> CMG drives for LEO spacecraft

Can Myxa be used for CMG drives? Should we mention this in addition to UAVs, UUVs and USVs in the "Overview" paragraph?

Comment (j3qq4hch) [reply docs/datasheet/Komar.tex:22]: > Komar has this additional sentence:
> 
> > Komar is based on the Mitochondrik control module, the hardware design of which is open source and available on Github.
> 
> Does this apply to Myxa too? Or is this unique to Komar?

This is unique to Komar.



> I understood that Komar does not have an accessible UART interface, so I have removed this. Is this correct?

This is correct

**Comment by clyde-johnston** [comment]
Closing pull request.  I will open a new one with the next commit.

**Pull request closed without merging**